docs/howtos/customization/setup_export_registry.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: "Setting Up An Export Destination Registry"
+sidebar_position: 8
+title: ""
+---
+
+:::note
+
+Do not confuse this topic with external Epinio registries.
+These are where Epinio saves the images of staged applications for its own use.
+See [Setting Up An External Container Registry](setup_external_registry.md) for more information.
+
+Export destination registries on the other hand are where a user saves active applications to,
+for pickup by and use with `helm` and other kubernetes tools.
+
+It does not help that it is perfectly ok to configure the same registry both as external registry
+and as export target.
+
+:::
+
+## How to set up a basic export destination registry
+
+As an operator, replace the placeholders in the command below, and then invoke it
+
+```bash
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/opaque
+metadata:
+  annotations:
+    epinio.io/registry-namespace: "placeholder-user"
+  labels:
+    epinio.io/api-export-registry: "true"
+  name: (placeholder-name)
```

Comment:
The command failed to me because this is different from the other placeholders:
 
`(placeholder-name)` vs `"placeholder-user"`

Reply:
changed with commit 9f34db541267bf0810f39aa05e7e09adbe81b3f4